server/apps/archive/archive.py

```diff
@@ -196,12 +195,14 @@ def on_update(self, updates, original):
             return
 
         if updates.get('publish_schedule'):
+
             if datetime.datetime.fromtimestamp(False).date() == updates.get('publish_schedule').date():
```

Comment:
While already changing this file, I would refactor this `False`to `0`, since semantically timestamps are integers.